ng2-components/ng2-alfresco-search/src/components/alfresco-search-autocomplete.component.ts

```diff
@@ -40,15 +40,29 @@ export class AlfrescoSearchAutocompleteComponent implements OnInit, OnChanges {
     @Input()
     ngClass: any;
 
+    @Input()
+    maxResults: number = 5;
+
     @Output()
     preview: EventEmitter<any> = new EventEmitter();
 
     @Output()
+    focusEmitter: EventEmitter<FocusEvent> = new EventEmitter<FocusEvent>();
```

Comment:
this will end with odd html markup as for me: `<element (focusEmitter)="handler">`
Can name be changed to something like `focus` or `onFocus` to be more html-friendly?